addons/project/static/src/project_sharing/chatter/composer_patch.js

```diff
@@ -1,8 +1,14 @@
 import { Composer } from "@mail/core/common/composer";
+import { useSuggestion } from "@mail/core/common/suggestion_hook";
 
 import { patch } from "@web/core/utils/patch";
 
 patch(Composer.prototype, {
+    setup() {
+        super.setup();
+        this.suggestion = useSuggestion();
+    },
```

Comment:
Why is this added specifically here in this PR?

Reply:
For other portal chatter use cases, we disabled suggestions (frontend only) as requested. But it was enabled for project sharing. Now that they are using a same folder, we need to explicitly enable it for project sharing because it's already disabled in the portal.

Reply:
But the base composer code (mail/core/common) already has this line, why add it again?

Reply:
`if (this.env.inFrontendPortalChatter) {` should be made in a way to only apply in non-project portal

Reply:
Since [this PR](https://github.com/odoo/odoo/pull/189794) `inFrontendPortalChatter` is set to true in project sharing chatter, which goes against the whole idea of having this parameter. The reason for this change in that PR was to remove the focus from the composer in project sharing. The correct fix should be to set the focus to false, as it was only there to keep the focus in the project sharing.

Reply:
PR #207455